build/template-restore-build-MSIdentityWeb.yaml

```diff
@@ -16,22 +16,20 @@ steps:
     platform: ${{ parameters.BuildPlatform }}
     configuration: ${{ parameters.BuildConfiguration }}
 
-- task: VSBuild@1
-  displayName: 'Build solution Microsoft.Identity.Web.sln'
+- task: DotNetCoreCLI@2
+  displayName: 'Build solution Microsoft.Identity.Web.sln and run tests'
   inputs:
-    solution: Microsoft.Identity.Web.sln
-    vsVersion: '16.0'
-    msbuildArgs: '/p:RunCodeAnalysis=false /p:MsIdentityWebSemVer=${{ parameters.MsIdentityWebSemVer }} /p:SourceLinkCreate=true'
-    platform: ${{ parameters.BuildPlatform }}
-    configuration: ${{ parameters.BuildConfiguration }}
-
+    command: build
```

Comment:
Can the tasks be combined into `command: build test`?

Reply:
no, we can't